src/components/member-card/index.js

```diff
@@ -45,17 +61,37 @@ const Card = ({ developerInfo, isOptionKey }) => {
       {dev && (
         <SuperUserOptions username={username} showSettings={showSettings} />
       )}
-      <motion.img
-        layoutId={username}
-        src={
-          isMember ? `${img_url}?${Math.random() * 100}` : '/images/Avatar.png'
-        }
-        onError={brokenImageHandler}
-        className={
-          isMember ? classNames.imgContainer : classNames.imgContainerNewMember
-        }
-        alt={username}
-      />
+
+      {isMember && (
+        <motion.img
+          layoutId={username}
+          src={
+            isMember
+              ? `${img_url}?${Math.random() * 100}`
+              : '/images/Avatar.png'
+          }
+          onError={brokenImageHandler}
+          className={
+            isMember
+              ? classNames.imgContainer
+              : classNames.imgContainerNewMember
+          }
+          alt={username}
+        />
+      )}
+
+      {!isMember && (
+        <div
+          className={classNames.new_member_img_absent_container}
+          ref={cardRef}
```

Comment:
why create refs, cant we just simply pass on the data and style into the element.
any reasoning?

Reply:
Good question,  In react we avoid accessing the dom directly, with useRef we can access the virtual dom elements, thats why. I hope it clears the doubt

Reply:
That i know but the question is why you need to access the dom, can't we just pass down as JavaScript expression in the jsx straight

Reply:
yeah looks like an overkill, thanks for the suggestion made the change